### ISSUE TITLE
For `list item` component very big sizes for text make component not readable

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#1607]
# Description

Version 3.0.0 will ship the `list item` component.
However, when plenty of elements are displayed ion the item, and if the user increases a lot the text sizes, the component is not readable.

# Suggestions

- If text size higher than 230%, change the layout. Leading element above, trailing element below

# Screenshot

<img height="500" alt="Image" src="https://github.com/user-attachments/assets/997144be-5930-4b12-9ed8-a121e566642d" />

https://github.com/user-attachments/assets/02f114ef-f6a5-47d5-8685-3a8f350b956c

**Comment by pylapp** [comment]
> [!CAUTION]
> **on hold** until #265 merged
